### PR TITLE
Add PDPD style autobroadcast

### PR DESCRIPTION
**Pull request by baojun-nervana**

### CODE REVIEW

File: src/ngraph/op/util/broadcasting.cpp [hunk 1]
```diff
@@ -172,6 +172,57 @@ static std::shared_ptr<ngraph::Node>
...
+/// \brief      Broadcast input node.
+///
+/// \param[in]  value         The input Node to be broadcast.
+/// \param[in]  output_shape  The output shape.
+/// \param[in]  axis          The start index to align with output_shape
+///
+/// \return     The broadcasted Node.
+///
+static std::shared_ptr<ngraph::Node> broadcast_node_pdpd_style(
+    const std::shared_ptr<ngraph::Node>& value, const ngraph::Shape& output_shape, int64_t axis)
```
[the remark this answers is not in the document]
Comment (baojun-nervana) [reply src/ngraph/op/util/broadcasting.cpp]: updated, thanks!

File: src/ngraph/partial_shape.cpp [hunk 2]
```diff
@@ -250,29 +250,79 @@ bool PartialShape::broadcast_merge_into(PartialShape& dst,
...
+                {
+                    return false;
+                }
+            }
+
+            return true;
+        }
+    }
+    break;
+    default: throw ngraph_error("Unsupported auto broadcast type");
```
Comment (aprocter) [inline src/ngraph/partial_shape.cpp]:
```suggestion
    default: NGRAPH_CHECK(false, "Unsupported auto broadcast type: ", autob.m_type);
```

File: README.md [hunk 3]
```diff
@@ -96,7 +91,7 @@ to improve it:
 [contrib guide]: https://www.ngraph.ai/documentation/contributing/guide
 [pull request]: https://github.com/NervanaSystems/ngraph/pulls
 [how to import]: https://www.ngraph.ai/tutorials/onnx-tutorial#import-a-model-with-onnx-and-ngraph
-[ngraph_wireframes_with_notice]: doc/sphinx/source/graphics/ngpipelines.png "nGraph wireframe"
+[ngraph_wireframes_with_notice]: doc/sphinx/source/graphics/nGraph_main.png "nGraph components"
```
Comment (dgkutnic) [inline README.md]: I don't think these changes are part of the PR, right?

Comment (baojun-nervana) [reply README.md]: No.

**Pull request merged**